miden-vm/src/internal.rs

```diff
@@ -264,7 +264,9 @@ impl InputFile {
 
     /// Parse a `Word` from a hex string.
     pub fn parse_word(word_hex: &str) -> Result<Word, String> {
```

Comment:
Theoretically, we should be able to replace this function with `Word::try_from()` - though, this would not support "incomplete strings" (i.e., the string would need to be 66 characters). Maybe it is worth updating how `Word::try_from()` works in `miden-crypto`? cc @huitseeker.